packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -39,10 +39,11 @@ import { WorkspaceService } from '@theia/workspace/lib/browser';
 import { FileResourceResolver } from '@theia/filesystem/lib/browser';
 import { SearchInWorkspaceResult, SearchInWorkspaceOptions } from '../common/search-in-workspace-interface';
 import { SearchInWorkspaceService } from './search-in-workspace-service';
+import { SearchInWorkspacePreferences } from './search-in-workspace-preferences';
 import { MEMORY_TEXT } from './in-memory-text-resource';
+import { FileUri } from '@theia/core/lib/node/file-uri';
```

Comment:
You are importing a library intended for `node` in the `browser`: that's a big no-no.